x/transfermiddleware/keeper/keeper.go

```diff
@@ -204,9 +207,14 @@ func (keeper Keeper) HasParachainIBCTokenInfoByAssetID(ctx sdk.Context, assetID
 	return store.Has(key)
 }
 
+// GetParachainIBCTokenInfoByNativeDenom implements types.TransferMiddlewareKeeper.
+func (keeper Keeper) GetParachainIBCTokenInfoByNativeDenom(ctx sdk.Context, nativeDenom string) pcktfrwdtypes.ParaChainIBCTokenInfo {
```

Comment:
New logic was introduced into func `GetParachainIBCTokenInfoByNativeDenom` but this function is not used anywhere. On top of it all usage of `GetParachainIBCTokenInfoByNativeDenom` replaced into other func as `ParachainIBCTokenInfoByNativeDenom`.
So it is better to move all logic inside ParachainIBCTokenInfoByNativeDenom into the old func GetParachainIBCTokenInfoByNativeDenom and do not replace the usage of old GetParachainIBCTokenInfoByNativeDenom